util/Hexagon/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+find_package(nlohmann_json)
+find_package(cxxopts)
```

Comment:
Is it actually useful have the `InstrumentedGenerator` target _without_ the `logcat2json` tool? If not, then I think these should both be `REQUIRED` and the `add_subdirectory(Hexagon)` call above should be guarded by an off-by-default `option(WITH_UTILS_HEXAGON ...)` instead.